c/src/ml-api-service-extension.c

```diff
@@ -1324,3 +1461,70 @@ ml_service_destroy (ml_service_h handle)
 
   return _ml_service_destroy_internal ((ml_service_s *) handle);
 }
+
+/**
+ * @brief Request the service related to machine-learning.
+ */
+int
+ml_service_request (ml_service_h handle, const char *config_path,
```

Comment:
Suggestion: same param with ml_service_add_input() 
ml_service_request (ml_service_h handle, const char *name, const ml_tensors_data_h data);

I think we can use common function for request.
case 1. use ml_service_set_information()
case 2. add param -json string (e.g., ml_service_request (ml_service_h handle, const char *name, const ml_tensors_data_h data, const char *json_option);)

Reply:
Nice!
Remove ml_sercice_add_input and changed to use ml_service_request ().

ml_service_request (ml_service_h handle, const char *name, const char *config_path, const ml_tensors_data_h data);